jujugui/static/gui/src/app/init/bundle-importer.js

```diff
@@ -673,6 +697,13 @@ class BundleImporter {
           annotations['gui-y'] = parseInt(annotations['gui-y']) + space;
         }
       }
+
+      if (extraAnnotations) {
```

Comment:
This if is not required [shrug].

Reply:
If you drag on the canvas the stuff within here throws an error as there's no 'additional annotations'.